packages/swingset-liveslots/test/virtual-objects/test-vo-real-gc.js

```diff
@@ -0,0 +1,74 @@
+/* global WeakRef */
+import test from 'ava';
+import '@endo/init/debug.js';
+
+import { Far } from '@endo/marshal';
+import { kser, kunser } from '../kmarshal.js';
+import { setupTestLiveslots } from '../liveslots-helpers.js';
+
+test('VO holds presence', async t => {
```

Comment:
Can the name of this test be improved?

Reply:
Yeah, it's not entirely coherent, it tests both that state gets written out by end-of-crank, and that the Representative gets dropped at the right times. I'll improve the name, but I need to come up with some additional tests. Mostly I was just trying to get the "real-gc" half of this pair established.